pyproject.toml

```diff
@@ -1,15 +1,188 @@
 [build-system]
 requires = [
+    "hatchling",
+    "hatch-conda",
+    "hatch-vcs",
+    "bleach",
+    "bokeh >=3.1.0,<3.2.0",
+    "cryptography <39",
+    "packaging",
     "param >=1.9.2",
     "pyct >=0.4.4",
-    "setuptools >=42",
-    "bokeh >=3.1.0,<3.2.0",
     "pyviz_comms >=0.7.4",
     "requests",
+    "tqdm >=4.48.0",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "panel"
+dynamic = ["version"]
+description = "A high level app and dashboarding solution for Python."
+readme = "README.md"
+license = { file = "LICENSE.txt" }
+requires-python = ">=3.8"
+authors = [
+    { name = "HoloViz", email = "developers@holoviz.org" },
+]
+maintainers = [
+    { name = "HoloViz", email = "developers@holoviz.org" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: BSD License",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Financial and Insurance Industry",
+    "Intended Audience :: Healthcare Industry",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: Legal Industry",
+    "Intended Audience :: Other Audience",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Office/Business",
+    "Topic :: Office/Business :: Financial",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "Topic :: Scientific/Engineering :: Visualization",
+    "Topic :: Software Development :: Libraries",
+]
+dependencies = [
     "bleach",
-    "packaging",
+    "bokeh >=3.1.0,<3.2.0",
+    "markdown",
+    "pandas >=1.2",
+    "param >=1.12.0",
+    "pyct >=0.4.4",
+    "pyviz_comms >=0.7.4",
+    "requests",
+    "setuptools >=42",
```

Comment:
This should be ready to be removed because of https://github.com/holoviz/panel/pull/4547 
```suggestion
```

Reply:
Removed.